src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
```

Comment:
You can use 
if ( ) {
} else if ( ) {
}

construction

Reply:
yes, i know. but in some issues it could create some problems. And that's why i don't use this construction

Reply:
@vadosik15 Why do u need to use `else if` after `return`?